packages/nx-monorepo/scripts/pnpm/link-bundled-transitive-deps.ts

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env ts-node
+/*! Copyright [Amazon.com](http://amazon.com/), Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0 */
+import * as path from "node:path";
+import * as child_process from "node:child_process";
+import { promisify } from "node:util";
+import * as fs from "fs-extra";
+import * as SemVer from "semver";
+
+interface Dependency {
```

Comment:
Can you just use: https://github.com/pnpm/pnpm/blob/main/reviewing/list/src/getPkgInfo.ts#L13 ?

Reply:
Done